### PR TITLE
Fix shoprite

### PR DESCRIPTION
**Pull request by jleedev**

### CODE REVIEW

File: locations/spiders/shoprite.py [hunk 1]
```diff
@@ -1,75 +1,41 @@
...
-                "city": city,
-                "state": state,
-                "postcode": post,
-                "website": "https://shoprite.com/store/" + store_id,
+                "ref": ref,
+                "website": f"https://www.shoprite.com/sm/planning/rsid/{ref}",
+                "name": store["name"],
+                "lat": store["location"]["latitude"],
+                "lon": store["location"]["longitude"],
+                "addr_full": store["addressLine1"],
```
Comment (Cj-Malone) [inline locations/spiders/shoprite.py:32]: Should this be street_address?

Comment (jleedev) [reply locations/spiders/shoprite.py:32]:
```suggestion
                "street_address": store["addressLine1"],
```

**Comment by Cj-Malone** [comment]
The opening hours aren't in OSM format, don't know if that's technically a blocker but ideally they would be parsed.

Otherwise LGTM.

DictParser would have done fairly good on this one, but you obviously don't have to use it.

**Review by Cj-Malone — APPROVED** [review]
LGTM, thank you

**Comment by iandees** [comment]
Thanks for working on the hours. Having consistent output + IDs is helpful to our downstream users so every bit helps.

**Pull request merged**